src/com/serotonin/mango/ScriptSessionAndUsers.java

```diff
@@ -0,0 +1,77 @@
+package com.serotonin.mango;
+
+import com.rits.cloning.Cloner;
+import com.serotonin.mango.vo.User;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.directwebremoting.*;
```

Comment:
Please use imports with an explicit declaration.

Reply:
ok, thank you